styles/_variables.scss

```diff
@@ -1,13 +1,13 @@
 // Variables
 
 // Colors
-$brand_color: #0072ef !default;
+$brand_color: rgba(23, 73, 77, 1) !default;
 $brand_text_color: #fff !default; //Color for text with an accent color background
 $text_color: #333 !default;
-$link_color: #0072ef !default;
+$link_color: rgb(23, 73, 77) !default;
 $background_color: #fff !default;
 
-$visited-link-color: zass-darken($link_color, 20%);
+$visited-link-color: $link_color;
```

Comment:
Does this mean we won't have a different color for visited links?

Reply:
Yes. This was asked for by design. CC @niksavic

Reply:
Can we consider keeping a different color? This functionality helps users understanding if they've already navigated to a link https://www.nngroup.com/articles/change-the-color-of-visited-links/